services/github/github-issue-detail.spec.js

```diff
@@ -99,9 +99,26 @@ describe('GithubIssueDetail', function() {
       which: 'state',
       json: { state: 'closed' },
     }).expect({
-      value: 'closed',
+      value: { state: 'closed', merged: true }
```

Comment:
the missing comma at the end of this line is the reason (or at least one of the reasons) why eslint, prettier and the unit tests are failing in the CI builds. 
```suggestion
      value: { state: 'closed', merged: true },
```